### ISSUE TITLE
dependencies

### ISSUE THREAD

**Issue by crackedbeefcake** [nathantreid/meteor-css-modules#60]
Yesterday out of nowhere my app stopped working.  it says `install.js: 101 Uncaught Error: Cannot find module './styles/new.sass'` in the console and when i restart meteor i get `App crashing'

```

Started MongoDB.
Errors prevented startup:

While loading plugin `mss` from package `nathantreid:css-modules`:
error: Can't load npm module 'autoprefixer' from packages/mss_plugin.js. Check your Npm.depends().
packages/mss/postcss-plugins.js:35:35: plugin autoprefixer was not found by NPM!
  at packages/mss/postcss-plugins.js:35:35
  at forEach (packages/ramda_ramda/upstream/dist/ramda.js:1874:1)
  at packages/ramda_ramda/upstream/dist/ramda.js:390:1
  at Object.f2 (packages/ramda_ramda/upstream/dist/ramda.js:434:1)
  at loadPlugins (packages/mss/postcss-plugins.js:32:4)
  at meteorInstall.node_modules.meteor.mss.postcss-plugins.js (packages/mss/postcss-plugins.js:21:16)
  at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
  at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
  at node_modules/meteor/mss/node_modules/recursive-readdir/node_modules/minimatch/node_modules/sigmund/sigmund.js:39:1
  at node_modules/meteor/mss/node_modules/recursive-readdir/node_modules/minimatch/node_modules/sigmund/sigmund.js:39:1


Your application has errors. Waiting for file change.
```

I reinstalled the package, updated meteor, and updated npm dependencies. I think its basically #57 again despite the fix

**Comment by akanix42** [comment]
Hmm, that's quite strange for it to happen with no changes. I haven't released #57 yet (that will likely be later today), so its currently only available through Github. However, #57 was an issue with importing a file in a local package from another file in that package.
For your issue it seems like Meteor can't find the autoprefixer plugin. Please try reinstalling `autoprefixer`:
In the app directory, `meteor npm install autoprefixer` (make sure you always precede any `npm` calls with `meteor` or you will very likely get errors).

If that doesn't work, please upload a repro to Github so I can take a look. Thanks!

**Comment by crackedbeefcake** [comment]
I installed `prefixer` and it somewhat worked. The error is gone, but none of the styles are loaded. 
I stripped my project down [and uploaded it here.](https://github.com/FinnFrotscher/css-modules-bugfix.git)

**Comment by crackedbeefcake** [comment]
`install.js:101Uncaught Error: Cannot find module './styles/new.sass'`
i still get this error. 
installed `autoprefixer` and all the `postCSS` plugins but it still doesnt work

**Comment by akanix42** [comment]
I wasn't able to reproduce the error. When I first ran your app, I received an error about node-sass, so I deleted the `node_modules` folder and ran `meteor npm install`.
After that the site loaded correctly, but the sass files weren't being transformed. This was due to the setting

```
    "postcssPlugins": {

    }
```

which runs postcss without any plugins.

I removed that setting from the package.json, which returns it to the default:

```
    "postcssPlugins": {
      "postcss-simple-vars": {},
      "postcss-modules-local-by-default": {},
      "postcss-modules-extract-imports": {},
      "postcss-modules-scope": {},
    }
```

 If you are still experiencing issues I recommend deleting the `.meteor/local` directory (**warning: this will wipe out your local database**) and your `node_modules` folder. Let me know how it goes!
